app/boards/shields/qaz/qaz.overlay

```diff
@@ -0,0 +1,48 @@
+#include <dt-bindings/zmk/matrix-transform.h>
+
+/ {
+    chosen {
+        zmk,kscan = &kscan0;
+        zmk,matrix_transform = &default_transform;
+        zmk,underglow = &led_strip;
```

Comment:
This should be added in the nice_nano overlay where the led_strip is defined. This will ensure this shield builds properly with other controllers like the BlueMicro840.